aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -209,53 +201,28 @@ mod test {
     //     |  700  |               |           | yes |         |
     //     |  800  |               |           |     |   yes   |
     //     +-------+---------------+-----------+-----+---------+
-    fn test_set_up(context: &NativeAggregatorContext, aggregator_enabled: bool) {
+    fn test_set_up(context: &NativeAggregatorContext) {
         let mut aggregator_data = context.aggregator_data.borrow_mut();
 
         aggregator_data.create_new_aggregator(aggregator_id_for_test(100), 100);
         aggregator_data.create_new_aggregator(aggregator_id_for_test(200), 200);
         aggregator_data.create_new_aggregator(aggregator_id_for_test(300), 300);
         aggregator_data.create_new_aggregator(aggregator_id_for_test(400), 400);
 
-        assert_ok!(aggregator_data.get_aggregator(
-            aggregator_id_for_test(100),
-            100,
-            context.resolver,
-            aggregator_enabled,
-        ));
-        assert_ok!(aggregator_data.get_aggregator(
-            aggregator_id_for_test(200),
-            200,
-            context.resolver,
-            aggregator_enabled,
-        ));
+        assert_ok!(aggregator_data.get_aggregator(aggregator_id_for_test(100), 100,));
```

Comment:
nit: trailing comma, otherwise not sure if the test logic still makes sense, but we can revisit later.

Reply:
Done.